app/presenters/chart_presenter.rb

```diff
@@ -42,6 +42,8 @@ def keys
   end
 
   def values
-    json[metric].map { |hash| hash['value'] }
+    json[metric].map do |hash|
+      metric == :satisfaction_score ? (hash['value'] * 100).round : hash['value']
```

Comment:
For the time being can we extract a helper to render the satisfaction score?

```ruby
json[metric].map { |hash| MetricsFormatterHelper.format(metric, hash['value']) }
```

We should be able to use it in other places as well.

Reply:
Also the helper can rely on Rails helper to render numbers / percentages.